### PR TITLE
Feature/48/admin

### PR DESCRIPTION
**Pull request by dboston528**
Added admin button and route that links to an admin page. The admin page renders a login page and list of users.

### CODE REVIEW

File: frontend/src/components/Login.jsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    render() {
+        return (
+            <div>
+                <h1>Young Masterbuilders in Motion</h1>
+                <form>
+                    <input type="text" placeholder="usernname" />
+                    <input type="password" placeholder="passwword" />
+                    <button>Log in</button>
+                </form>
+
```
Comment (mkirlin) [inline frontend/src/components/Login.jsx]: Can you delete these empty lines?

File: frontend/src/components/SiteUser.jsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+                <li>
+                    <div>
+                        <p>{user.name}</p>
+                        <p>{user.email}</p>
+                        <button>Delete User</button>
+                    </div>
+                </li>
+            ))}
+        </ol>
+
```
Comment (mkirlin) [inline frontend/src/components/SiteUser.jsx]: Please also delete these empty lines.

File: frontend/src/components/SiteUser.jsx [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+function SiteUser(props) {
+    return (
+        <ol>
+            {props.goof.map((user) => (
```
Comment (mkirlin) [inline frontend/src/components/SiteUser.jsx]: What is `.goof` doing here?

File: frontend/src/pages/admin.jsx [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+    render() {
+        return (
+            
+                    
+            <div>
+                < Header />    
+                <h1>Young Masterbuilders in Motion</h1>
+                <form>
+                    <input type="text" placeholder="usernname" />
```
Comment (mkirlin) [inline frontend/src/pages/admin.jsx]: There're typos on the placeholders in both of these lines.

File: frontend/src/pages/admin.jsx [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+                    <input type="password" placeholder="passwword" />
+                    <button>Admin</button>
```
Comment (mkirlin) [inline frontend/src/pages/admin.jsx]: What is this button actually doing?

File: frontend/src/pages/admin.jsx [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+                </form>
+                <SiteUser goof={this.state.user} />
```
Comment (mkirlin) [inline frontend/src/pages/admin.jsx]: Ahhh, here's where the goof is coming from. Yeah, let's change that.

File: frontend/src/pages/admin.jsx (hunk 5 above)
Comment (dboston528) [reply frontend/src/pages/admin.jsx]: We have an admin login page and a regular login page. We can get rid of it if you want.

File: frontend/src/pages/admin.jsx (hunk 6 above)
Comment (dboston528) [reply frontend/src/pages/admin.jsx]: Changed.

File: .gitignore [hunk 7]
```diff
@@ -12,6 +12,8 @@
 
 .env
 
+Pipfile
```
Comment (mkirlin) [inline .gitignore]: This is unnecessary; we just shouldn't have this file in the top level directory at all.

File: Pipfile [hunk 8]
```diff
@@ -0,0 +1,11 @@
+[[source]]
```
[the remark this answers is not in the document]
Comment (mkirlin) [reply Pipfile]: This is still showing up in the PR, so it's still going to go into staging/master if we merge.

File: frontend/src/components/Header/index.js [hunk 9]
```diff
@@ -2,8 +2,16 @@ import React, { Component } from 'react';
 import { NavLink } from 'react-router-dom';
 import './Header.css';
 
+const Navigation = () => (
+  <nav className='navbar navbar-expand-lg navbar-light bg-light'>
+    
+        {/* {authUser => (authUser ? <HeaderAdmin/> : <HeaderNonAuth />)} */}
```
Comment (mkirlin) [inline frontend/src/components/Header/index.js]: Please delete this commented-out line.

File: frontend/src/components/Header/index.js [hunk 10]
```diff
@@ -2,8 +2,16 @@ import React, { Component } from 'react';
...
+        <HeaderAdmin/>
+  </nav>
+);
 
-class Header extends Component {
+
+class HeaderNonAuth extends Component {
```
Comment (mkirlin) [inline frontend/src/components/Header/index.js]: Let's just rename this to Header

File: node_modules/.yarn-integrity [hunk 11]
```diff
@@ -0,0 +1,20 @@
+{
```
Comment (mkirlin) [inline node_modules/.yarn-integrity]: The entire node_modules directory needs to be deleted.

File: package.json [hunk 12]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (mkirlin) [inline package.json]: This needs to be deleted.

File: yarn.lock [hunk 13]
```diff
@@ -0,0 +1,35 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (mkirlin) [inline yarn.lock]: This needs to be deleted.

File: Pipfile (hunk 8 above)
[the remark this answers is not in the document]
Comment (dboston528) [reply Pipfile]: Deleted

**Pull request merged**